conbench/db.py

```diff
@@ -132,6 +136,7 @@ def create_all():
         else:
             raise
 
+    log.info("create_all(engine) returned. dispose()")
```

Comment:
Of the many many `make run-app-dev` I execute locally, sometimes the stack startup hangs in `create_all()`. This log statement helps in the future to better understand where the hang is; but for now I think I have an educated guess that it was a hanging connect() -- the timeout plus retrying will help.